client/src/components/Header.tsx

```diff
@@ -95,7 +98,27 @@ function AccountMenu() {
           'aria-labelledby': 'menu-btn',
         }}
       >
-        <MenuItem onClick={closeMenu}>My Posts</MenuItem>
+        <MenuItem
+          onClick={async () => {
+            try {
+              const result = await api.me();
```

Comment:
Doesn't the current user info exist in context to be shared across the application? We shouldn't need to hit `me` again.

@plausibly

Reply:
Oh yea, that is true. On `App.tsx` there is `UserContext`